post_installation_configuration/connected-to-disconnected.adoc

```diff
@@ -0,0 +1,66 @@
+[id="connected-to-disconnected"]
+= Converting a connected cluster to a disconnected cluster
+include::modules/common-attributes.adoc[]
+:context: connected-to-disconnected
+
+toc::[]
+
+
+
+There might be some scenarios where you need to convert your {product-title} cluster from a connected cluster to a disconnected cluster.  
+
+A disconnected cluster, also known as a restricted cluster, does not have an active connection to the internet. As such,you must mirror the contents of your registries and installation media. You can create this mirror registry on a host that can access both the internet and your closed network, or copy images to a device that you can move across network boundaries.
+
+This topic describes the general process for converting an existing, connected cluster into a disconnected cluster.    
+
+:FeatureName: The process for converting a cluster from conncted to disconnected 
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
+
+[id="prerequisites_connected-to-disconnected"]
+== Prerequisites
+
+* The `oc` client is installed.
+
+* A running cluster.
+
+* An installed mirror registry, which is a container image registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2] in the location that will host the {product-title} cluster, such as one of the following registries:
++
+--
+** link:https://www.redhat.com/en/technologies/cloud-computing/quay[Red Hat Quay]
+
+** link:https://jfrog.com/artifactory/[JFrog Artifactory]
+
+** link:https://www.sonatype.com/products/repository-oss?topnav=true[Sonatype Nexus Repository]
+
+** link:https://goharbor.io/[Harbor]
+--
++
+If you have an entitlement to Red Hat Quay, see the documentation on deploying Red Hat Quay link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html/deploy_red_hat_quay_for_proof-of-concept_non-production_purposes/[for proof-of-concept purposes] or link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html/deploy_red_hat_quay_on_openshift_with_the_quay_operator/[by using the Quay Operator].
```

Comment:
I am not entirely clear on when to use _entitlement_ over _subscription_ as detailed in the SSG. I am just calling this out here to verify your usage.